### DISCUSSION TITLE
cr0sh-unblocker

### DISCUSSION THREAD

**Discussion by HelloIloveGimkit2** [3kh0/ext-remover#1154] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
https://helloilovegimkit2.github.io/cr0sh-unblocker/

DON'T SAY THE REGULAR CR0SH WORD BECAUSE IT'S BLOCKED (replace the o with a 0)

**Comment by S-PScripts** [comment]
crosh :trollface:

**Reply by HelloIloveGimkit2** [reply]
oh wait
I think it's only if it's in the url
